DokuwikiXMLExport.php

```diff
@@ -59,7 +59,8 @@ public function generateXMLExport($start, $count)
         }
         $exporter = Exporter::create(Exporter::TYPE_XML, $count);
 
-        $total = count($pages);
+        // total -1 because the XML starts counting at 0 like an array
+        $total = count($pages) - 1;
```

Comment:
Are you sure? If there is 1 page in the wiki then I guess `$pages == 1`. The `total` attribute in the XML should then also be 1.